### ISSUE TITLE
Generalization With Neuro-NER and overfiting disease

### ISSUE THREAD

**Issue by Parvez-Khan-1** [Franck-Dernoncourt/NeuroNER#85]
Hello Frank,
I used Neuro-ner to solve one of my problem , it working fine but looks like it suffered with over fitting disease of ML.It showing 99.85% on train_set , 100.00% on validation_set and 82.73% on test_set.
I need your suggestion to get rid of this problem so that my model is a generalized model which will perform good on unseen data also, looking forward to hear from you.

Thanks
ParvezKhan

**Comment by grafael** [comment]
Increasing dropout values may help you.

**Comment by Parvez-Khan-1** [comment]
If i increase dropout rate so, should i also need to increase learning rate?. Currently i have 0.5 drop out rate and 0.5 learning rate.

**Comment by grafael** [comment]
No, as far as I know, changing the learning rate will not prevent the overfitting.

**Comment by JohnGiorgi** [comment]
Hi ParvezKhan,

In order to address over-fitting, there is really two major things you can do: __regularization__ (increase dropout) and __get more data__.

Is there anyway you can gather more data for your task? Or perhaps, use your trained model to tag some unlabeled text and __bootstrap__ the process like that.
